### PR TITLE
Add csdk-v1 CI devnet, end test collisions

### PR DESCRIPTION
**Pull request by mhagel**
## Link to Issue
Closes: #3506 
Closed: #3441 

## Description of Changes
- Spins up a Cosmos devnet from a docker image every CI run. Eliminates collisions.
- Adds `csdk-v1` community to db, specifically for CI tests

## Test Plan
- Devnet tests pass

## Other Considerations

-

### CODE REVIEW

File: .github/workflows/CI.yml
Comment (mhagel) [inline .github/workflows/CI.yml:1]: @NakulManchanda Is this what you had in mind?

File: packages/chain-events/cosmos-chain-testing/start.sh [hunk 1]
```diff
@@ -12,4 +12,4 @@ if [ "$1" = "--build" ]; then
     cd $SCRIPT_DIR
     docker build -t ${IMAGE_NAME} .
 fi
-docker run -p 5050:5050 -e PORT=${PORT} --name ${IMAGE_NAME} ${IMAGE_NAME}
\ No newline at end of file
+docker run -p 5050:5050 -e PORT=${PORT} --name ${IMAGE_NAME} ${IMAGE_NAME} & START_PID=$!
```
Comment (mhagel) [inline packages/chain-events/cosmos-chain-testing/start.sh:15]: adding this was key to running the chain in the background so the next CI step could run. But let me know if there is a better way. The way it is now, it takes a long time to build and run the devnet from scratch.

@NakulManchanda

File: .github/workflows/CI.yml
Comment (NakulManchanda) [reply .github/workflows/CI.yml:1]: yes! exactly

File: packages/chain-events/cosmos-chain-testing/start.sh (hunk 1 above)
Comment (NakulManchanda) [reply packages/chain-events/cosmos-chain-testing/start.sh:15]: didn't understand this, but rest looks right!

Comment (NakulManchanda) [reply packages/chain-events/cosmos-chain-testing/start.sh:15]: yeah we can start in background, if we don't want to wait for image to download everytime!
cool, thats fine, thanks!

**Review by NakulManchanda — APPROVED** [review]

**Comment by NakulManchanda** [comment]
what was the issue with caching?

File: packages/chain-events/cosmos-chain-testing/start.sh (hunk 1 above)
Comment (NakulManchanda) [reply packages/chain-events/cosmos-chain-testing/start.sh:15]: - this helps locally only, script doesn't get called via CI.yml at all
- other option is use `-d` detached mode
```
docker run -p 5050:5050 -e PORT=${PORT} --name ${IMAGE_NAME} -d
```

We kept it this way for local to see streaming logs in terminal during startup to spot errors if any, we can always stream like this later
```
docker logs -f ${IMAGE_NAME}
```

**Comment by NakulManchanda** [comment]
> the active proposal cache is set at 5 minutes. That broke the first test because the second request doesn’t pick up the newly created proposal.

will add cache control header to disable caching when not required. It will help with tests!
We don't need to cache in integration/unit tests.
This is one such case, don't want to cache to introduce bugs in integration level caching.

```
Cache-Control: no-cache
```

**Review by jnaviask — APPROVED** [review]

**Comment by NakulManchanda** [comment]
- based on discussion with @mhagel 
- Proposal etc kind of work little differently - we go through these LCDClient etc, which don't give us a chance to add custom headers, but some others who use chai.request or axios to make api request in their test will benefit from `Cache-Control` header

**Pull request merged**